### DISCUSSION TITLE
Duplicate poll for repeating appointments

### DISCUSSION THREAD

**Discussion by ionarn** [lukevella/rallly#443] [Ideas]
Hey, first I want to thank the dev team for developing and maintaining this amazing tool.
I weekly organize board game groups and use rallly to find a day in the week that fits for most people.

The idea:
Because I create a poll weekly, I copy-paste all the text from a previous poll to a new one.
A _duplicate poll_ option would save a bit of time in such use cases.
What do you think about that?

Thank you!
ionarn

**Comment by lukevella** [comment]
Definitely a highly requested feature and it's been on the backlog for some time. It's just been pushed back as it's not a must-have feature for most, but I will create an issue from this to keep it visible.

**Comment by xiaoveiti** [comment]
+1 would be awesome - especially if have every month the same poll 🤣 

I just notice there is an option, but there is no chance to just transfer it to another month. Just duplicating the name and the description. if you can also transfer the days for example from february to march - that would be awesome!
